lib/karafka/process.rb

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
+
 module Karafka
   # Class used to catch signals from ruby Signal class in order to manage Karafka shutdown
   # @note There might be only one process - this class is a singleton
   class Process
     include Singleton
 
     # Signal types that we handle
-    HANDLED_SIGNALS = %i(
+    HANDLED_SIGNALS = %i[
```

Comment:
Unfortunately, slightly different; it should also restart everything. If we want to get this out the door I can just get that out in a sep. PR sometime later on this week/weekend?